src/app/activity/bounty/List.ts

```diff
@@ -28,145 +27,131 @@ export const listBounty = async (request: ListRequest): Promise<any> => {
         customerId: request.guildId,
     });
 
-    const channel = await listUser.guild.channels.fetch(dbCustomerResult.bountyChannel);
-    const channelName = channel.name;
-
-
     Log.debug('Connected to database successfully.');
```

Comment:
This debug log is a past mistake of mine (confusing when parsing our application logs). If you get the chance would you mind removing it for me?